redbox-core/redbox/loader/ingester.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from typing import TYPE_CHECKING
+
+from langchain_core.runnables import RunnableParallel
+from langchain_elasticsearch.vectorstores import BM25Strategy, ElasticsearchStore
+
+from redbox.chains.components import get_embeddings
+from redbox.chains.ingest import ingest_from_loader
+from redbox.loader.loaders import UnstructuredChunkLoader
+from redbox.models.chain import GeneratedMetadata
+from redbox.models.settings import get_settings
+from redbox.models.file import ChunkResolution
+
+if TYPE_CHECKING:
+    from mypy_boto3_s3.client import S3Client
+else:
+    S3Client = object
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+env = get_settings()
+alias = env.elastic_chunk_alias
+
+
+def get_elasticsearch_store(es, es_index_name: str):
+    return ElasticsearchStore(
+        index_name=es_index_name,
+        embedding=get_embeddings(env),
+        es_connection=es,
+        query_field="text",
+        vector_query_field=env.embedding_document_field_name,
+    )
+
+
+def get_elasticsearch_store_without_embeddings(es, es_index_name: str):
+    return ElasticsearchStore(
+        index_name=es_index_name,
+        es_connection=es,
+        query_field="text",
+        strategy=BM25Strategy(),
+    )
+
+
+def create_alias(alias: str):
+    es = env.elasticsearch_client()
+
+    chunk_index_name = alias[:-8]  # removes -current
+
+    es.options(ignore_status=[400]).indices.create(index=chunk_index_name)
+    es.indices.put_alias(index=chunk_index_name, name=alias)
+
+
+def _ingest_file(file_name: str, es_index_name: str = alias):
+    logging.info("Ingesting file: %s", file_name)
+
+    es = env.elasticsearch_client()
```

Comment:
What does this look like without elasticsearch?

Reply:
like this https://github.com/gecBurton/dosac/blob/c86f04daf96daea7b18b6ee431e9ef1e8c03ba3f/core/models.py#L101 (without the embeddings)?